feat: add updated migration guide doc (#930)
novuhq/docs: content/docs/platform/inbox/migration-guide.mdx

---

<Callout type="info">

>   - Use [@novu/react](/platform/sdks/react) Inbox react component and

Reviewer: ```suggestion
  - Use [@novu/react](/platform/sdks/react) Inbox React component and
```

>   [@novu/js](/platform/sdks/javascript) headless package with new dashboard workflows and [@novu/framework](/platform/workflow/overview) based workflows.

Reviewer: ```suggestion
  [@novu/js](/platform/sdks/javascript) headless package with new dashboard workflows and [@novu/framework](/platform/workflow/overview)-based workflows.
```

>    - With old dashboard based workflows, use

Reviewer: What do you mean by "old" here? Can you give me a date or some way a user knows their workflow is old in this context?

export default Novu;
```

## Notification center without bell icon

> The @novu/react package introduces a flexible way to display notifications as a list without the default bell icon. Utilize the `Inbox` and `Notifications` components to achieve this functionality.

Reviewer: ```suggestion
The @novu/react package introduces a flexible way to display notifications as a list without the default bell icon. Use the `Inbox` and `Notifications` components to achieve this functionality.
```

export default Novu;
```

## Avatar icons

> In old implementation, you could set the avatar icon for a notification by toggle on the `Add an avatar` option in the workflow UI. Novu will use the avatar field of actor subscriber for avatar icon.

Reviewer: ```suggestion
In the old implementation, you could set the avatar icon for a notification by toggle on the `Add an avatar` option in the workflow UI. Novu uses the avatar field of the actor subscriber for avatar icon.
```

> In new implementation, you can set the avatar icon for a notification by adding the avatar icon in the workflow UI. There are 3 options to choose from:

Reviewer: ```suggestion
In the new implementation, you can set the avatar icon for a notification by adding the avatar icon in the workflow UI. There are three options to choose from:
```

> - Use a default avatar icon

Reviewer: ```suggestion
- Use a default avatar icon.
```

> - Use hard coded avatar icon url

Reviewer: ```suggestion
- Use a hard-coded avatar icon URL.
```

> - Use payload variable to dynamically set the avatar icon

Reviewer: ```suggestion
- Use a payload variable to dynamically set the avatar icon.
```

## Popover positioning

> For advanced positioning and styling of the notifications popover, integrate third-party popover libraries such as Radix UI.

Reviewer: Do not use "popover." Use "dialog" instead. Apply throughout.

For more information on `appearance` customization visit [here](/platform/inbox/react/styling).

## Multiple tabs support

Organize notifications into different categories using tabs by leveraging the tags property in workflow definitions and the tabs prop in the Inbox component.

### Create multiple tabs

> - Old Implementation

Reviewer: ```suggestion
- Old implementation
```

export default Novu;
```

## HMAC encryption

> The process remains the same as before. See the detailed guide [here](/platform/inbox/prepare-for-production#secure-your-inbox-with-hmac-encryption).

Reviewer: ```suggestion
The process remains the same as before. For more information, refer to [Secure your inbox with HMAC encryption](/platform/inbox/prepare-for-production#secure-your-inbox-with-hmac-encryption).
```
